frontend/public/locales/en/common.json

```diff
@@ -1,8 +1,7 @@
 {
   "Index": {
-    "title": "Home",
-    "intro": "This is a template for a React web application using the <LinkToNextJs>Next.js framework</LinkToNextJs>.",
-    "body": "This is template includes:<ul><li>Framework for server-side rendered, static, or hybrid React applications</li><li>TypeScript and React testing tools</li><li>U.S. Web Design System for themeable styling and a set of common components</li><li>Type checking, linting, and code formatting tools</li><li>Storybook for a frontend workshop environment</li></ul>",
+    "page_title": "Home | Beta.grants.gov",
+    "meta_description": "The beta version of a government website where federal agencies post discretionary funding opportunities and grantees find and apply for them",
```

Comment:
🤔 Let's use some already-approved copy. 

Maybe repeat the phrase from `goal_paragraph_2`: "A one-stop shop for all federal discretionary funding to make it easy for you to discover, understand, and apply for opportunities."

Or the last sentence in `goal_paragraph_1`: "Our mission is to increase access to grants and improve the grants experience for everyone."

Reply:
Why not both?

Reply:
If I have to pick, I'd go with the first. The first description is more clear/specific about what the beta site offers. The second is more around the broader mission and goal. For SEO reasons, I'd go with the first because of the keywords (discover, apply, federal discretionary funding) which might be what people are searching.

Reply:
Great. I'll go with the sentence `A one-stop shop for all federal discretionary funding to make it easy for you to discover, understand, and apply for opportunities.` and consider that as decided.